client/src/components/Patient/PatientDashboard.tsx

```diff
@@ -15,19 +17,27 @@ const PatientDashboard: FunctionComponent<{}> = () => {
     console.log(await fetch('POST', '/verify_token/', {}));
   };
 
-  const appointmentHandler = () =>{
-    navigate('/patientappointments')
+  const appointmentHandler = () => {
+    navigate('/patientappointments');
+  };
+  useEffect(() => {
+    fetchDoctors();
 
-  }
+    //getAppointment();
+  }, []);
+  const fetchDoctors = async () => {
+    await fetchAllDoctors?.('0x8eda1014b9177d464306935e8fcf9fd27c20aa08');
```

Comment:
why is the address hardcoded?

Reply:
The integration is a work in progress. This will be updated once the others code is merged.